### DISCUSSION TITLE
Newcomers use the opai-API to ask for help!!!!!!!!!!

### DISCUSSION THREAD

**Discussion by jia-fie** [community/community#123268] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

i'm using openai_API always eroor（:openai.APIConnectionError:Connection error.），However, you can go to the official website。What's wrong with this?I've tried the methods on the Internet almost all over again，Hope to answer

**Comment by jia-fie** [comment]
beg beg big guy，help me

**Comment by ebndev** [comment]
Hi @jia-fie, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
